schema/: merge Project and Location extensions

odscjen opened the pull request:
closes #1484 

was unable to run manage.py, got the following error

> Exception: project is multilingual at planning/budget/project, but not at planning/project

which I think is because `planning/budget/project` is a string field and `planning/project` is an object.

odscjen committed:
schema/: merge Project and Location extensions

odscjen committed:
update changelog.md

odscjen commented:
@jpmckinney does an exception need to be added to `manage.py` to allow `planning.project` to be nonmultilingual?

odscjen committed:
add Location and Project to examples release_package.json and schema/reference

jpmckinney commented:
> @jpmckinney does an exception need to be added to manage.py to allow planning.project to be nonmultilingual?

I'll make the necessary changes once the PR is ready for review.

odscjen committed:
update guidance linked_standards and add link to OC4IDS identifiers guidance

odscjen committed:
release-schema: Project and Location changes in line with style guide

odscjen commented:
Thanks @jpmckinney this is ready for review now

odscjen committed:
Merge branch '1.2-dev' into 1484-merge-project-and-location

jpmckinney committed:
build: Update manage.py for planning.project being an object

jpmckinney committed:
build: Run ./manage.py pre-commit

odscjen committed:
Merge branch '1.2-dev' into 1484-merge-project-and-location

odscjen commented:
spell check false positive, picking up on "sme"

jpmckinney committed:
build: Update script to keep planning/budget/project as multilingual, but not coordinates. ci: Allow SME acronyms.

jpmckinney committed:
Review merger of Project and Location extensions

- Merge reference.md content into schema.
- Copy-edit Project and Location subschema.
- Copy-edit Location codelists. Remove normative word imported from RFC 7946.
- Use classification scheme example that conforms to guidance.
- Use "datasets" not "data sets" per style guide.

jpmckinney committed:
geometryType.csv: Reduce repetition

jpmckinney committed:
release-schema: Restore some clarity to budget.projectID

jpmckinney committed:
build: Run manage.py pre-commit

jpmckinney committed:
release-schema: Align budget.project with projectID. If we're saying "as it appears in a budget", then we don't need to furthermore refer to a registry.

jpmckinney reviewed (approved):
I made several changes. Please review. Mainly:

- Put information in the schema (which we know people read) rather than the docs (which we know people read less frequently).
- Tell people what to do, rather than tell them what not to do. This meant making budget.projectID clearer.
- Remove sentences that didn't have a lot of content.
- Don't repeat GeoJSON norms.

jpmckinney committed:
build: Run manage.py pre-commit

jpmckinney committed:
guidance/map/linked_standards: Clarify direction of OC4IDS link

odscjen commented on schema/release-schema.json:
```diff
@@ -2736,6 +2883,82 @@
+        },
+        "title": {
+          "title": "Title",
+          "description": "The name of the project. If a projects register is available, this should be the same as the project's name in that register.",
```
```suggestion
          "description": "The name of the project. If a project's register is available, this should be the same as the project's name in that register.",
```

odscjen committed:
Update schema/release-schema.json

odscjen committed:
build: run manage.py precommit

odscjen commented:
1 small typo that I've just gone ahead and corrected. Otherwise all changes look good

jpmckinney commented:
I'm not sure that was a typo? We typically talk about a projects register (register of projects) not an (individual) "project's register".

odscjen commented:
oops, sorry I'd misread the sentence! I'll do a quick PR to undo that change rather than trying to rollback seeing as the branch has already been deleted

odscjen commented:
ah I see you've fixed that elsewhere, thank you!
